README.md

```
@@ -169,20 +220,26 @@ sudo mkdir -p /usr/local/bin
 sudo cp .build/release/RemindersSync /usr/local/bin/obsidian-reminders
 sudo cp .build/release/ScanVault /usr/local/bin/obsidian-scan
 sudo cp .build/release/ExportOtherReminders /usr/local/bin/obsidian-export
+sudo cp .build/release/ReSyncReminders /usr/local/bin/obsidian-resync
+sudo cp .build/release/CleanUp /usr/local/bin/obsidian-cleanup
 ```
 
 3. Make them executable:
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 3; Style: 1/1/1

(MD029, ol-prefix)